test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -61,7 +61,7 @@ _load_kind_cluster() {
 
 _install_tools() {
   # install yq
-  if [ -z "$(which yq)" ]; then
+  if [ -z "$(command -v yq)" ]; then
```

Comment:
How was this error not caught in the other PR?

Reply:
We have fixed shellcheck command in https://github.com/kubeflow/katib/pull/1885 and https://github.com/kubeflow/katib/pull/1884.
But I forgot to rebase https://github.com/kubeflow/katib/pull/1882 after merging their PRs.

So, we could not catch this error in https://github.com/kubeflow/katib/pull/1882.